package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "Magic-Mirror-Module-Traffic",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "Google Maps Traffic module for the Magic Mirror.",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/SamLewis0602/MMM-Traffic"
+    "url": "git+https://github.com/nigel-daniels/MMM-Traffic"
```

Comment:
This shouldn't change, as the core project will still be located on my page. It could continue to evolve after your contribution, just as it has after previous users' contributions, and my page is where all of the changes are consolidated.